src/NOTD.SC2Map/MapScript.galaxy

```diff
@@ -6660,6 +6661,29 @@ bool auto_gf_ECEnvironmentalChanges_TriggerFunc (bool testConds, bool runActions
     return true;
 }
 
+void gf_ECShopXPReq () {
+    // Variable Declarations
+    bool lv_vets;
+
+    // Variable Initialization
+    lv_vets = true;
+
+    // Implementation
+    PlayerGroupLoopBegin(gv_marineSquad);
+    for ( ; !PlayerGroupLoopDone() ; PlayerGroupLoopStep() ) {
+        if ((PlayerGetPropertyInt(PlayerGroupLoopCurrent(), c_playerPropCustom) < 5000)) {
+            lv_vets = false;
+        }
+
+    }
+    PlayerGroupLoopEnd();
+    if ((lv_vets == false)) {
+        UnitSetInfoText2(UnitFromId(178), StringExternal("Param/Value/F64869B1"));
+        UnitSetInfoText2(UnitFromId(2503), StringExternal("Param/Value/97BA6CAF"));
```

Comment:
Check all players to see if anyone is under 5000 xp. If any player is under 5000 xp (in Easy Company non-recruit modes), the XP requirement will be displayed to the Troop Transport and the Pipedream's Emporium. This is intended for "first exposure" to vet games for low xp players, which typically takes place in Easy Company.